README.md

```diff
@@ -107,17 +108,24 @@ accounts available between them.*
 
 ### Full node on the Rinkeby test network
 
-The above test network is a cross-client one based on the ethash proof-of-work consensus
-algorithm. As such, it has certain extra overhead and is more susceptible to reorganization
-attacks due to the network's low difficulty/security. Go Ethereum also supports connecting
-to a proof-of-authority based test network called [*Rinkeby*](https://www.rinkeby.io)
-(operated by members of the community). This network is lighter, more secure, but is only
-supported by go-ethereum.
+Go Ethereum also supports connecting to the older proof-of-authority based test network 
+called [*Rinkeby*](https://www.rinkeby.io) which is operated by members of the community.
 
 ```shell
 $ geth --rinkeby console
 ```
 
+### Full node on the Ropsten test network
+
+In addition to Görli and Rinkeby, Go-Ethereum also supports the ancient Ropsten testnet.
```

Comment:
A general naming convention: `go-ethereum` (all lowercase) is the repo/project and `Geth` (capitalized) is the thing that people run. In this particular case you can use either, but if you choose the former, please keep it lowercase and ideally put it between ticks.

A bit of a special flake is if you want to start a sentence with `go-ethereum`, because then lowercase looks weird. In those cases we do use `Go Ethereum` capitalized without the dash, but generally try to keep those constructs to the minimum.